targets/ChibiOS/_nf-overlay/os/hal/ports/RP/LLD/RTCv2/hal_rtc_lld.c

```
@@ -0,0 +1,510 @@
+ * @brief   Set alarm time.
+ * @note    The alarm time can be partially specified by leaving fields as
+ *          zero and excluding them from the mask.
+ * @note    The function can be called from any context.
+ *
+ * @param[in] rtcp      pointer to RTC driver structure.
+ * @param[in] alarm     alarm identifier. Can be 0.
+ * @param[in] alarmspec pointer to a @p RTCAlarm structure.
+ *
+ * @notapi
+ */
+void rtc_lld_set_alarm(RTCDriver *rtcp, rtcalarm_t alarm, const RTCAlarm *alarmspec)
```

Cppcheck (2.21.0)

[style] The function 'rtc_lld_set_alarm' is never used.

(unusedFunction)